Client/Ecosia/UI/NTP/NTPLayout.swift

```diff
@@ -36,7 +36,7 @@ class NTPLayout: UICollectionViewFlowLayout {
                     let font = UIFont.preferredFont(forTextStyle: .callout)
                     let height = text.heightWithConstrainedWidth(width: impact.bounds.width - 16 - 48, font: font) + 24 + 16
                     tooltip.frame.size.height = height
-                    tooltip.frame.origin.y -= (height - 32)
+                    tooltip.frame.origin.y -= (height - 40)
```

Comment:
I wonder if we would benefit from static constant declaration for these values (40). Same for line 37, for example, where there are operations like .width -16 -48, maybe having them as constants would increase readability?